### ISSUE TITLE
[Bug]: Can't stream with Android

### ISSUE THREAD

**Issue by gokugen** [apivideo/api.video-reactnative-live-stream#62]
### Version

v1.2.3

### Which operating systems have you used?

- [X] Android
- [ ] iOS

### Environment that reproduces the issue

Samsung Galaxy S9 Android 10

### Is it reproducible in the example application?

Yes

### RTMP Server

rtmp://192.168.1.38:1935/live

### Reproduction steps

- Start streaming with some bitrate (2 * 1024 * 1024 or 3*1024 * 1024 or another value)
- Set the resolution to 1080p or 720p
- Set the gopDuration between 2-4s
- After some time streaming will stop - server will not receive data
- Preview will still work fine on the phone
- Change the bitrate value, gopDuration, resolution and test again

### Expected result

Streaming don't stop

### Actual result

Streaming stop after a short period (1s to 20s depending on the configuration) of time because of a wrong configuration

**Comment by ThibaultBee** [comment]
Hi,

Thanks for reporting this issue. Could you add logcat trace?

**Comment by gokugen** [comment]
Yes of course, here it is [logcat.txt](https://github.com/apivideo/api.video-reactnative-live-stream/files/10965516/logcat.txt)

Another thing related to this is that I need sometimes to call stopStreaming and then startStreaming again in order to see my streaming on my server but as I said above my streaming stops after a few seconds anyway.

**Comment by ThibaultBee** [comment]
All I can find is
```
rtmpdroid: Can't connect stream
```

`connectStream` is a RTMP message that is exchanged between the client and the server once they are connected. The issue is often because the `streamKey` is not correctly set. Could you double check it?

If your stream key is okay, could you please contact your server to check what is wrong with the `connectStream` message?

**Comment by gokugen** [comment]
The `streamKey` is good and I can see my stream info on my NodeMediaServer but without the audio and video.

<img width="1920" alt="Screenshot 2023-03-14 at 10 53 09" src="https://user-images.githubusercontent.com/14259676/224963841-7943a24a-472a-4d8c-bd7e-64b46a6aac74.png">

But when I call `stopStreaming` and then `startStreaming` again the stream shows up on my server but stops after few seconds.

<img width="1920" alt="Screenshot 2023-03-14 at 10 54 17" src="https://user-images.githubusercontent.com/14259676/224963897-b7fe8a28-2f7d-4a8a-a797-2f3cc46da877.png">

#### Here is another [logcat.txt](https://github.com/apivideo/api.video-reactnative-live-stream/files/10966986/logcat.txt) after I called stopStreaming and startStreaming again.

**Comment by ThibaultBee** [comment]
Hi,

Sorry for the delay, I could not reproduce this with my phone.
Could you check if you can stream to nms with Could you check with https://github.com/ThibaultBee/StreamPack/suites/18441384366/artifacts/1069203816? (internal Android RTMP library)?

**Comment by ThibaultBee** [comment]
This is supposed to be fixed with 2.0.0.
Could you test with 2.0.0?

**Comment by ThibaultBee** [comment]
Stall issue.
Please reopen if you have new info.
